### PR TITLE
Amend margins on mobile displays

### LINKED ISSUE #655: Margins are too narrow on mobile
This is not a high priority issue, but compared with the old theme, the left/right margins of a blog post are too narrowly close to the edges of the screen on mobile, especially on the 21:9 aspect ratio screens etc...

You can see the old margins pre ##624:

IMG_0101

And after:

IMG_0100

I have not seen any actual text being cropped just yet, but it'd be a good idea to have some small `margin` distance from the display's edges.

- **Plume version:** latest commit
- **Related issues:** #650
- **Operating system:** iOS 12/iPhone X, but I've also tested it on Android 9
- **Web Browser:** iOS Safari, and also Firefox and Chrome for Android

### PR DESCRIPTION
**Pull request by marek-lach**
Fixes #655 and also most of #604.

**Edit:** Better addressed by #663. Closing this issue now.

### CODE REVIEW

**Comment by AnaGelez** [comment]
Looks like it is too much now…

image

image

**Comment by AnaGelez** [comment]
Still the same for me… :confused: I can try to have a look tomorrow or on Saturday if you want.

**Comment by marek-lach** [comment]
I'll see what I can do...

**Comment by marek-lach** [comment]
@AnaGelez All tested and seems to work well on my end.

**Comment by AnaGelez** [comment]
The article content is indeed better (maybe even a bit too much), but some elements that shouldn't have margins now have (article cover/heading, and all the info at the bottom from the license to the comment area). :confused:

**Comment by marek-lach** [comment]
> The article content is indeed better (maybe even a bit too much), but some elements that shouldn't have margins now have (article cover/heading, and all the info at the bottom from the license to the comment area). 😕

All of these issues should be ok now.

**Comment by marek-lach** [comment]
Not really sure why the article cover image isn't fully scaled to be honest...

**Comment by marek-lach** [comment]
> I felt you were struggling with CSS here

You are right. It's a lot of guess work for me, I am not really natural at it :-)

File: assets/themes/default/_forms.scss [hunk 1]
```diff
@@ -144,7 +144,7 @@ form.new-post {
 
   & > * {
     flex-grow: 1;
-    max-width: 40%;
+    max-width: 60%;
```
Comment (marek-lach) [inline assets/themes/default/_forms.scss]: This bit here seems to make the _article meta_ less squashed than before on small screens, I think?

Before:
Screenshot 2019-09-01 at 19 23 09

After #662:

Screenshot 2019-09-01 at 19 23 44
 
May be worth replicating in #663 if you want...

Comment (AnaGelez) [reply assets/themes/default/_forms.scss]: I'm not sure these lines apply on *article meta* section, as it doesn't have the `split` class too (it is used only for the license/tags part, not the rest). But I have something similar to that in my PR. Thanks anyway!

**Pull request closed without merging**